From 16e0ca8119b99b244c9eeafd78bb9eb43e4ba639 Mon Sep 17 00:00:00 2001
From: Aaron Kimball <aaron@cloudera.com>
Date: Fri, 12 Mar 2010 14:40:20 -0800
Subject: [PATCH 0036/1179] MAPREDUCE-703. Sqoop requires dependency on hsqldb in ivy

Description: Sqoop builds crash without explicit dependency on hsqldb.
Reason: build system bugfix
Author: Aaron Kimball
Ref: UNKNOWN
---
 src/contrib/sqoop/ivy.xml |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/contrib/sqoop/ivy.xml b/src/contrib/sqoop/ivy.xml
index cf58d6a..8fe13c4 100644
--- a/src/contrib/sqoop/ivy.xml
+++ b/src/contrib/sqoop/ivy.xml
@@ -56,5 +56,9 @@
       name="log4j"
       rev="${log4j.version}"
       conf="common->master"/>
+    <dependency org="hsqldb"
+      name="hsqldb"
+      rev="${hsqldb.version}"
+      conf="common->default"/>
     </dependencies>
 </ivy-module>
-- 
1.7.0.4

